src/command/cmd_funcs.c

```diff
@@ -188,44 +188,41 @@ _string_matches_one_of(const char* what, const char* is, bool is_can_be_null, co
     return ret;
 }
 
-/*
- * Take a line of input and process it, return TRUE if profanity is to
- * continue, FALSE otherwise
+/**
+ * @brief Processes a line of input and determines if profanity should continue.
+ *
+ * @param window
+ * @param inp The input string to process.
+ * @return Returns TRUE if profanity is to continue, FALSE otherwise.
```

Comment:
I don't see what advantage this comment has over the one we already had.
It's just more lengthy.

Reply:
Re doxygen please see https://github.com/profanity-im/profanity/issues/1859